web/data/data_test.go

```diff
@@ -668,3 +668,25 @@ func TestUnsupportedOption(t *testing.T) {
 	assert.Equal(t, "400 Bad Request", res.Status, "should get a 400")
 	assert.NoError(t, err)
 }
+
+func TestGetAllDocs(t *testing.T) {
+	url := ts.URL + "/data/" + Type + "/_all_docs?include_docs=true"
+	req, _ := http.NewRequest("GET", url, nil)
+	req.Header.Add("Host", Host)
+	out, res, err := doRequest(req, nil)
+	assert.Equal(t, "200 OK", res.Status, "should get a 200")
+	assert.NoError(t, err)
+	totalRows := out["total_rows"].(float64)
+	assert.Equal(t, float64(3), totalRows)
```

Comment:
Why do you cast to `float64`? I first thought it could be some requirement of the assert package, but looking at the doc, it doesn't seem so... Just out of curiosity :)